.github/workflows/build-plugin-zip.yml

```diff
@@ -71,7 +71,7 @@ jobs:
             - name: Checkout code
               uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
               with:
-                  token: ${{ secrets.GUTENBERG_TOKEN }}
```

Comment:
I will revert this change before merging (from `GITHUB_TOKEN` -> `GUTENBERG_TOKEN`).